app/navigation/LoadingNavigator.scala

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package navigation
+
+import com.google.inject.Singleton
+import models._
+import pages.Page
+import pages.loading._
+import play.api.mvc.Call
+
+@Singleton
+class LoadingNavigator {
+
+  protected def normalRoutes(departureId: String, mode: Mode): PartialFunction[Page, UserAnswers => Option[Call]] = {
+    case AddUnLocodeYesNoPage         => ua => addUnlocodeNormalRoute(ua, departureId)
+    case UnLocodePage                 => ua => CountryPage.route(ua, departureId, NormalMode)
```

Comment:
I think this should be go to the AddExtraInformationYesNo page - it's optional whether they add country/location